angular-workspace/package.json

```diff
@@ -60,6 +60,6 @@
     "ng-packagr": "^16.2.3",
```

Comment:
Re-built the lockfile because it was getting crufty with lots of overrides. i.e. a  clean repo with npm install was creating overrides in like every project node_modules:

```
C:\dev\github\nimble>git clean -fdx
Removing angular-workspace/node_modules/
Removing node_modules/
Removing packages/jasmine-parameterized/node_modules/
Removing packages/nimble-blazor/build/generate-playwright-version-properties/dist/
Removing packages/nimble-blazor/node_modules/
Removing packages/nimble-components/node_modules/
Removing packages/nimble-tokens/node_modules/
Removing packages/performance/node_modules/
Removing packages/site/node_modules/
Removing packages/xliff-to-json-converter/node_modules/
```

After rebuilding the lock file a clean npm install created much fewer overrides:
```
C:\dev\github\nimble>git clean -fdx
Removing node_modules/
Removing packages/nimble-blazor/build/generate-playwright-version-properties/dist/
Removing packages/nimble-blazor/node_modules/
Removing packages/nimble-tokens/node_modules/
```